x/cronos/keeper/evm_log_handlers_test.go

```diff
@@ -346,10 +353,10 @@ func (suite *KeeperTestSuite) TestSendCroToIbcHandler() {
 				balance := suite.app.BankKeeper.GetBalance(suite.ctx, sdk.AccAddress(types.ModuleName), types.IbcCroDenomDefaultValue)
 				suite.Require().Equal(coin, balance)
 				ibcCoin := sdk.NewCoin(types.IbcCroDenomDefaultValue, sdk.NewInt(123))
-				ibcBalance := suite.app.BankKeeper.GetBalance(suite.ctx, sdk.AccAddress(contract.Bytes()), types.IbcCroDenomDefaultValue)
+				ibcBalance := suite.app.BankKeeper.GetBalance(suite.ctx, sdk.AccAddress(sender.Bytes()), types.IbcCroDenomDefaultValue)
```

Comment:
I don'g quite get this assertion, aftet `NewSendCroToIbcHandler`, isn't it that the IBC token on sender is supposed to 
1. Burn by IBC module
2. Waiting for relayer to pickup
So the IBC balance of the sender should be 0?

Reply:
yeah that is correct but in this test, I am using a mock IBC module, so that is why the balance remains

Reply:
Gotcha

Reply:
Maybe good to add a comment in this case